librz/include/rz_il/rz_il_opbuilder_begin.h

```diff
@@ -52,6 +52,42 @@
 #define S32(val)   SN(32, val)
 #define S64(val)   SN(64, val)
 
+#define BV2F(fmt, bv)            rz_il_op_new_float(fmt, bv)
+#define F32(f32)                 rz_il_op_new_float_from_f32(f32)
+#define F64(f64)                 rz_il_op_new_float_from_f64(f64)
+#define F2BV(fl)                 rz_il_op_new_fbits(fl)
+#define IS_FINITE(fl)            rz_il_op_new_is_finite(fl)
+#define IS_FNAN(fl)              rz_il_op_new_is_nan(fl)
+#define IS_FINF(fl)              rz_il_op_new_is_inf(fl)
+#define IS_FZERO(fl)             rz_il_op_new_is_fzero(fl)
+#define IS_FNEG(fl)              rz_il_op_new_is_fneg(fl)
+#define IS_FPOS(fl)              rz_il_op_new_is_fpos(fl)
+#define FNEG(fl)                 rz_il_op_new_fneg(fl)
+#define FABS(fl)                 rz_il_op_new_fabs(fl)
+#define F2INT(l, rmode, fl)      rz_il_op_new_fcast_int(l, rmode, fl)
+#define F2SINT(l, rmode, fl)     rz_il_op_new_fcast_sint(l, rmode, fl)
+#define INT2F(fmt, rmode, bv)    rz_il_op_new_fcast_float(fmt, rmode, bv)
+#define SINT2F(fmt, rmode, bv)   rz_il_op_new_fcast_sfloat(fmt, rmode, bv)
+#define FCONVERT(fmt, rmode, fl) rz_il_op_new_fconvert(fmt, rmode, fl)
+// TODO: FEQUAL?
+#define FSUCC(fl)                  rz_il_op_new_fsucc(fl)
+#define FPRED(fl)                  rz_il_op_new_fpred(fl)
+#define FORDER(flx, fly)           rz_il_op_new_forder(flx, fly)
+#define FROUND(rmode, fl)          rz_il_op_new_fround(rmode, fl)
+#define FSQRT(rmode, fl)           rz_il_op_new_fsqrt(rmode, fl)
+#define FRSQRT(rmode, fl)          rz_il_op_new_fsucc(rmode, fl)
+#define FADD(rmode, flx, fly)      rz_il_op_new_fadd(rmode, flx, fly)
+#define FSUB(rmode, flx, fly)      rz_il_op_new_fsub(rmode, flx, fly)
+#define FMUL(rmode, flx, fly)      rz_il_op_new_fmul(rmode, flx, fly)
+#define FDIV(rmode, flx, fly)      rz_il_op_new_fdiv(rmode, flx, fly)
+#define FMOD(rmode, flx, fly)      rz_il_op_new_fdiv(rmode, flx, fly)
+#define FHYPOT(rmode, flx, fly)    rz_il_op_new_fhypot(rmode, flx, fly)
+#define FPOW(rmode, flx, fly)      rz_il_op_new_fpow(rmode, flx, fly)
+#define FMAD(rmode, flx, fly, flz) rz_il_op_new_fmad(rmode, flx, fly, flz)
+#define FROOTN(rmode, flx, n)      rz_il_op_new_frootn(rmode, flx, n)
+#define FPOWN(rmode, flx, n)       rz_il_op_new_fpown(rmode, flx, n)
+#define FCOMPOUND(rmode, flx, n)   rz_il_op_new_fcompund(rmode, flx, n)
+
```

Comment:
```suggestion
#define FLOATV16(bv)       rz_il_op_new_float(RZ_FLOAT_IEEE754_BIN_16, bv)
#define FLOATV32(bv)       rz_il_op_new_float(RZ_FLOAT_IEEE754_BIN_32, bv)
#define FLOATV64(bv)       rz_il_op_new_float(RZ_FLOAT_IEEE754_BIN_64, bv)
#define FLOATV128(bv)      rz_il_op_new_float(RZ_FLOAT_IEEE754_BIN_128, bv)

#define FSUCC(f)                       rz_il_op_new_fsucc(f)
#define FPRED(f)                       rz_il_op_new_fpred(f)
```
we could add those also